tests/jerry/es.next/math-cbrt.js

```diff
@@ -29,5 +29,5 @@ assert(Math.cbrt(Number.NEGATIVE_INFINITY) === Number.NEGATIVE_INFINITY);
 assert(Math.cbrt(1.0) === 1.0);
 assert(Math.cbrt(-1.0) === -1.0);
 
-assert(Math.cbrt(27.0) === 3.0);
+// assert(Math.cbrt(27.0) === 3.0); // FIXME: unstable, depending on compiler and libm
```

Comment:
How much unsable it is? do we need specialize cbrt?

Reply:
This is not a test262 failure, but a failure in jerry's own test suite.
As far as I could track it down, this a floating point numeric instability issue, perhaps rounding on the least significant bit (3.000...x != 3.000...0). Observed with gcc + libm + -fno-builtins. Does not reproduce with clang. (thanks to @galpeter)